e2e-tests/src/test/java/gov/nasa/jpl/aerie/e2e/procedural/scheduling/AutoDeletionTests.java

```diff
@@ -0,0 +1,137 @@
+package gov.nasa.jpl.aerie.e2e.procedural.scheduling;
+
+import gov.nasa.jpl.aerie.e2e.types.GoalInvocationId;
+import gov.nasa.jpl.aerie.e2e.utils.GatewayRequests;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import javax.json.Json;
+import java.io.IOException;
+import java.util.Objects;
+import java.util.concurrent.atomic.AtomicReference;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class AutoDeletionTests extends ProceduralSchedulingSetup {
+  private GoalInvocationId edslId;
+  private GoalInvocationId procedureId;
+
+  @BeforeEach
+  void localBeforeEach() throws IOException {
+    try (final var gateway = new GatewayRequests(playwright)) {
+      final String coexGoalDefinition =
+          """
+          export default function myGoal() {
+            return Goal.CoexistenceGoal({
+              forEach: ActivityExpression.ofType(ActivityTypes.BiteBanana),
+              activityTemplate: ActivityTemplates.GrowBanana({quantity: 1, growingDuration: Temporal.Duration.from({minutes:1})}),
+              startsAt:TimingConstraint.singleton(WindowProperty.START).plus(Temporal.Duration.from({ minutes : 5}))
+            })
+          }""";
+
+      edslId = hasura.createSchedulingSpecGoal(
```

Comment:
Minor, nonblocking: Could you add a comment somewhere about how this goal is being used to test if the plan is empty? I want to make sure that we don't lose this understanding if we ever have to refactor these tests.